00-RELEASENOTES

```diff
@@ -1,16 +1,146 @@
+* Free strings during BGSAVE/BGAOFRW to reduce copy-on-write (#905)
+* Creates an empty lua table with specified initial capacity as much as possible (#1092)
+* Move prepareClientToWrite out of loop for HGETALL command (#1119)
+* Improved hashing algorithm in luaS_newlstr (#1168)
+* Replace dict with new hashtable for sets datatype (#1176)
+* Do security attack check only when command not found to reduce the critical path. (#1212)
+* Accelerate hash table iterator with prefetching (#1501)
+* Replace dict with new hashtable: hash datatype (#1502)
+* Replace dict with new hashtable for sets datatype (#1176)
+* Replace dict with new hashtable: sorted set datatype (#1427)
+* Trim free space from inline command argument strings to avoid excess memory usage (#1213)
+* Increase the IO_THREADS_MAX_NUM. (#1220)
+* Refactor of ActiveDefrag to reduce latencies (#1242)
+* Integrating fast_float to optionally replace strtod (#1260)
+* Improvements for TLS with I/O threads (#1271)
+* Optimize PFCOUNT, PFMERGE command by SIMD acceleration (#1293)
+* Optimize sdscatrepr by batch processing printable characters (#1342)
+* Optimize ZRANK to avoid path comparisons (#1389)
+* move clientCron onto a separate timer (#1387)
+* client struct: lazy init components and optimize struct layout (#1405)
+* Offload reading the replication stream to IO threads (#1449)
+* skip CRC checksumming during diskless full sync with TLS enabled. (#1479)
+* Accelerate hash table iterator with value prefetching (#1568)
+
+New/Modified configurations
+===========================
+* Add io-threads-do-reads config to deprecated config table to have no effect. (#1138)
+
+Build and Packaging changes
+=================================
+* RDMA builtin support (#1209)
+* Fix Valkey binary build workflow, version support changes. (#1429)
+
+Code improvements
+=================
+* Remove valkey specific changes in jemalloc source code (#1266)
+
+Module Improvements
+===================
+* Use `DEEPBIND` flag when loading external modules using `dlopen` (#1703)
+* Add support for MustObeyClient Module API (#1582)
+* Adds support for scripting engines as Valkey modules (#1277)
+
+Cluster improvements
+====================
+* Do election in order based on failed primary rank to avoid voting conflicts (#1018)
+* Make replica CLUSTER RESET flush async based on lazyfree-lazy-user-flush (#1190)
+* Trigger the election as soon as possible when doing a forced manual failover (#1067)
+* Make manual failover reset the on-going election to promote failover (#1274)
+* Brocast a PONG to all node in cluster when role changed (#1295)
+* Manual failover vote is not limited by two times the node timeout (#1305)
+* Automatic failover vote is not limited by two times the node timeout (#1356)
+
+Behavior Changes
+================
+* Rax size tracking (#688)
+* Take hz into account in activerehashing to avoid CPU spikes (#977)
+* Incr expired_keys if the expiration time is already expired (#1517)
```

Comment:
Incr in what kind o stats? Keyspace notification?? I don't remember but it might be good to mention here.

Reply:
Yes It made changes to the keyspace notifications. I can elaborate some more